jennchenn/uwexplore
# Fetch schedule ID from user login

## Implementation description

- retrieve schedule ID once user logs in and display the appropriate schedule
- currently just refreshing everytime the user revisits the page so essentially they shouldn't get logged out unless they manually clear local storage? [resolves BREAD-146]
- added in logout button [resolves BREAD-106]
- removed the lorem ipsum login text [resolves BREAD-145]



## Steps to test

1. Login w valid email
2. Verify schedule updates
3. Refresh page - verify you are still able to see courses
4. Logout
5. Verify you are logged out (empty calendar!)



## What should reviewers focus on?

- this is kinda messy oops but will help us be able to test w more than one person tomorrow? also my first frontend ticket 🥺
